src/e2eFlutterTest/java/io/appium/java_client/android/BaseFlutterTest.java

```diff
@@ -46,16 +49,20 @@ public static void beforeClass() {
 
     @BeforeEach
     public void startSession() throws MalformedURLException {
+        FlutterDriverOptions flutterOptions = new FlutterDriverOptions()
+                .setFlutterSystemPort(9999)
+                .setFlutterServerLaunchTimeout(Duration.ofSeconds(10));
+
         if (IS_ANDROID) {
-            // TODO: update it with FlutterDriverOptions once implemented
             UiAutomator2Options options = new UiAutomator2Options()
```

Comment:
I find it confusing that we still need to create UiAutomator2Options instance. 

Lets have FlutterDriverOptions classes that contains flutter-specific options and inside compose xcuitest and uia2 driver options. Then we could call it like

```
FlutterIosOptions fio = new FlutterDriverOptions()
  .setFlutterSystemPort(9999)
  .setXcuitestOptions(.....)
driver = new FlutterIOSDriver(service.getUrl(), fio);
```